File: frontend/src/app/models/RoboAdvisor.ts
export class RoboAdvisor {
  constructor(
    public name: string,
    public price: number,
    public quantity: number,
    public priceDiff: number,
    public quantDiff: number,
    public option: string,
    public risk: number,
    public returnRate: number,
    public score: number
  ) {}
}

File: spring-capstone-tradehub/src/main/java/com/fidelity/business/RoboAdvisor.java
package com.fidelity.business;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.FieldDefaults;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@FieldDefaults(makeFinal = false, level = lombok.AccessLevel.PRIVATE)
public class RoboAdvisor {

    String name;
    int price;
    int quantity;
    int priceDiff;
    int quantDiff;
    String option;
    double risk;
    double returnRate;
    double score;

}